docs/publishing.md

```diff
@@ -16,74 +16,43 @@
 
 > Steps for publishing the monorepo
 
-1. Make sure your local branch is up-to-date by running
-   `git pull upstream master`
-2. Run `yarn sync` to make sure all packages are synced
-3. Make sure your git status is clean (will also be checked in our publish
-   task), if not commit and push changes, then go to Step 1
-4. Make sure dependencies are up-to-date by doing `yarn clean` && `yarn install`
-5. Run `yarn build` to build all package assets
-6. Run `yarn lerna version` with the appropriate flags (see release steps below)
+## Prerelease
 
-## Pre-release
+Often, are prereleases are when we switch between minor versions of the system. To do this, we'll need to follow the following steps:
```

Comment:
Should this say "Often, _our_ prereleases are..."?

Reply:
Oh duh lol. Thanks!